### DISCUSSION TITLE
changing Traefik options in development

### DISCUSSION THREAD

**Discussion by nareto** [fastapi/full-stack-fastapi-template#1827] [Questions] [answered]
I would like to use the Traefik buffering middleware for the backend to limit the size of the incoming requests. However, no matter how I change the labels in `docker-compose.yml`, it doesn't seem to affect Traefik (at least what it shows in the dashboard). This is what I have now (relevant part of `docker-compose.yml`):

    proxy:
        image: traefik:v2.2
        deploy:
          labels:
            - traefik.http.middlewares.${STACK_NAME?Variable not set}-limit.buffering.maxRequestBodyBytes=500000
    backend:
      deploy:
        labels:
          - traefik.http.routers.${STACK_NAME?Variable not set}-backend-http.middlewares=${STACK_NAME?Variable not set}-limit

However, after stopping the proxy container and running `docker-compose up -d`, the middleware does not show up in the Traefik UI. 

BTW I would like these changes to be present ALSO when I deploy.

**Comment by paxcodes** [comment, marked as answer]
If you need it in development, you'd also have to put it in the `labels` prop of your container in `override.yml`.

`deploy` prop is only used when you deploy your stack.
